lesson_25/AutoProject/Auto/admin.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib import admin
+from Auto.models import Car, CarModel, User, Brand
+
+
+class CarModelInline(admin.TabularInline):
+    model = CarModel
+
+
+class BrandInline(admin.TabularInline):
+    model = Brand
+
+
+class CarInline(admin.TabularInline):
+    model = Car
+
+
+@admin.register(Brand)
+class BrandAdmin(admin.ModelAdmin):
+    ordering = ('name',)
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_display = ('get_car_full_name', 'vin', 'is_active')
+    search_fields = ('vin', )
+
+    def get_car_full_name(self, obj):
+        return f"{obj.car_brand.name} {obj.car_model.name}"
+    get_car_full_name.short_description = 'Car'
```

Comment:
what is it?

Reply:
This is so that in the list of cars, each car has a brand and model, as people most often represent them, for example: This is a bmw x7, it has a broken engine.